### PR TITLE
refactor metrics defaultCollector&&agg

### PR DESCRIPTION
**Pull request by wxbty**
## What is the purpose of the change
Use the same API as the three centers
link https://github.com/apache/dubbo/issues/11941

### CODE REVIEW

**Comment by wxbty** [comment]
@AlbumenJ @songxiaosheng  PTAL

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java [hunk 1]
```diff
@@ -56,4 +73,120 @@ public static Map<String, String> serviceTags(String appAndServiceName) {
...
+        if (e.isBiz()) {
+            targetKey = MetricsKey.METRIC_REQUEST_BUSINESS_FAILED;
+        }
+        if (e.isSerialization()) {
+            targetKey = MetricsKey.METRIC_REQUESTS_CODEC_FAILED;
+        }
+        if (e.isNetwork()) {
+            targetKey = MetricsKey.METRIC_REQUESTS_NETWORK_FAILED;
+        }
+        return targetKey;
```
Comment (songxiaosheng) [inline dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:106]: in the future we also cloud use  map to  store  the  relation exception to metric key

**Review by songxiaosheng — APPROVED** [review]
LGTM

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java (hunk 1 above)
Comment (wxbty) [reply dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:106]: 0

**Pull request merged**